pkgs/applications/editors/emacs/make-emacs.nix

```diff
@@ -337,6 +338,7 @@ mkDerivation (finalAttrs: {
   ++ lib.optional withTreeSitter "--with-tree-sitter"
   ++ lib.optional withXinput2 "--with-xinput2"
   ++ lib.optional withXwidgets "--with-xwidgets"
+  ++ lib.optional (withSmallJaDic && lib.versionAtLeast version "29") "--with-small-ja-dic"
```

Comment:
I think it's better to remove `lib.versionAtLeast version "29"` from here and just blindly apply what was passed instead.

Reply:
I was wondering whether there'd be any `configure` errors otherwise, but now i see that things like `--with-tree-sitter` are also present here (and it seems like Emacs just emits a warning if a `--with-*`-style `configure` option is not found)